src/index.html

```diff
@@ -4,14 +4,207 @@
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="input">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            tabindex="6"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="input">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            id="pass"
+            name="pass"
+            tabindex="7"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="input">
+          Do you love cats?
+          <input
+            type="radio"
+            id="catsY"
+            name="cats"
+            value="y"
+            tabindex="8"
+          >
+          <label for="catsY">Yes</label>
+          <input
+            type="radio"
+            id="catsN"
+            name="cats"
+            value="n"
+            tabindex="9"
+          >
+          <label for="catsN">No</label>
+        </div>
+
+        <div class="input">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            id="color"
+            name="color"
+            tabindex="10"
+          >
+        </div>
+
+        <div class="input">
+          <label for="timeGoBed">What time do you go to bed?</label>
+          <input
+            type="time"
+            id="timeGoBed"
+            name="tomeGoBed"
+            tabindex="11"
+          >
+        </div>
+
+        <div class="input">
+          <label for="Car">What are your favorite brand of cars?</label>
+          <select
+            name="Car"
+            id="Car"
+            multiple
+            tabindex="12"
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="input">
+          <label for="rateWork">How are you rate our work?</label>
+          <input
+            type="range"
+            id="rateWork"
+            name="rateWork"
+            tabindex="13"
+            value="0"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <div class="input">
+          <label for="comm">Comments:</label>
+          <textarea
+            name="comm"
+            id="comm"
+            cols="15"
+            rows="2"
+            tabindex="14"
+          >
+          </textarea>
+        </div>
+
+        <div class="input">
+          <label for="reccom">Would you recommend us?</label>
+          <select
+            name="reccom"
+            id="reccom"
```

Comment:
Don't try to make shorten values like this, it's bad practice
```suggestion
            name="recommend"
            id="reccommmend"
```